src/v0/destinations/facebook_pixel/transform.js

```diff
@@ -200,6 +204,42 @@ const handleProduct = (message, categoryToContent, valueFieldIdentifier) => {
   };
 };
 
+const handleSearch = (message) => {
```

Comment:
can not find any use of `valueFieldIdentifier` but it is used while calling the function

Reply:
updated.